privacy-protections/fingerprinting/helpers/constants.js

```diff
@@ -0,0 +1,9 @@
+// fingerprint.js + various other sources
```

Comment:
I've put all the boring and long stuff in this file to keep others a bit more redable.